tools/manifest/manifest.py

```diff
@@ -1,26 +1,40 @@
 import itertools
 import json
 import os
-from collections import defaultdict
-from six import iteritems, iterkeys, itervalues, string_types
+from collections import MutableMapping, defaultdict
+from six import iterkeys, iteritems, itervalues, string_types
 
 from . import vcs
-from .item import (ManualTest, WebDriverSpecTest, Stub, RefTestNode, RefTest,
+from .item import (ManifestItem, ManualTest, WebDriverSpecTest, Stub, RefTestNode, RefTest,
                    TestharnessTest, SupportFile, ConformanceCheckerTest, VisualTest)
 from .log import get_logger
 from .utils import from_os_path, to_os_path
+from .sourcefile import SourceFile
 
 MYPY = False
 if MYPY:
     # MYPY is set to True when run under Mypy.
+    from logging import Logger
+    from typing import Any
+    from typing import Container
     from typing import Dict
-    from types import ModuleType
+    from typing import IO
+    from typing import Iterable
+    from typing import Iterator
+    from typing import List
+    from typing import Optional
+    from typing import Set
+    from typing import Text
+    from typing import Tuple
+    from typing import Type
+    from typing import Union
+    from typing import cast
 
 try:
     import ujson
-    fast_json = ujson  # type: ModuleType
+    fast_json = ujson
 except ImportError:
-    fast_json = json
+    fast_json = json  # type: ignore
```

Comment:
Why?

Reply:
Because `json` and `ujson` aren't the same.